docs/companies/index.html

```diff
@@ -160,6 +160,11 @@ <h2 class="lead">Utah Companies Using Go in Production</h2>
       <td>Lehi</td>
       <td></td>
     </tr>
+    <tr>
```

Comment:
The `docs` folder is auto generated from the build. Since this is not invasive, I won't require you to move it out, but it's generally not necessary to manually build, as that will happen on redeploy. Just a FYI